backend/README.md

```diff
@@ -105,3 +135,5 @@ Below are some exceptions and additions to the guidelines.
 
 - Prefer to not use named return values. Return values should be obvious from the context, their types, and finally from the function/method comment.
 - Don't use naked returns. They make the code harder to read and understand.
+
+
```

Comment:
These too are probably unintended changes. Please remove these empty lines, but so that the README ends on an empty line, i.e. no change is shown in this PR.